### PR TITLE
fix home docs page button on mobile & tablet

### PR DESCRIPTION
**Pull request by Arhell**
#19040

### CODE REVIEW

**Comment by jimangel** [comment]
It **does** appear to fix the linked issue (tested on Pixel 4 XL w/ firefox app). Any input from localization teams as far as impact? We could also only merge the English fix and allow localization to pick up the change as they update.

/cc @kubernetes/sig-docs-zh-reviews @kubernetes/sig-docs-ko-reviews @kubernetes/sig-docs-de-reviews @kubernetes/sig-docs-es-reviews @kubernetes/sig-docs-pt-reviews @kubernetes/sig-docs-id-reviews @kubernetes/sig-docs-ru-reviews @kubernetes/sig-docs-pl-reviews @kubernetes/sig-docs-vi-reviews

**Comment by Arhell** [comment]
@jimangel 
I agree with you, replaced only for **en** localization

**Comment by mfilocha** [comment]
>  We could also only merge the English fix and allow localization to pick up the change as they update.


+1 for @kubernetes/sig-docs-pl-reviews

**Comment by sftim** [comment]
Hi @Arhell 

To review this, I'll want to check that the code changes match the PR description. Could you summarize what changes this PR makes and (briefly) why those are useful?
If you can provide before and after links, that will make reviewing even easier.

**Comment by Arhell** [comment]
@sftim 
look issue please, the button on mobile did not work there, I hid it
Before:
https://kubernetes.io/docs/home/
after:
https://deploy-preview-19646--kubernetes-io-master-staging.netlify.com/docs/home/

and screen
before: 

after:



+ it did not work on this page due to the fact that there is no content for this button, which brings up a menu using an example
https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/

**Comment by rlenferink** [comment]
Removing incorrect labels as localization teams will update in follow-up PRs. Also this PR looks good to me. I'll leave approval to @sftim.

/lgtm

/remove-language de
/remove-language es
/remove-language fr
/remove-language id
/remove-language ja
/remove-language ko
/remove-language pl
/remove-language pt
/remove-language ru
/remove-language vi
/remove-language zh
